spec/03-plugins/28-grpc-gateway/01-proxy_spec.lua

```diff
@@ -202,5 +202,35 @@ for _, strategy in helpers.each_strategy() do
       assert.equal(400, res.status)
     end)
 
+    describe("regression", function()
+      test("empty array in json #FTI-5002", function()
```

Comment:
let's not use jira ticket number here, it's not visible to community and if we move to another jira project the number got missing. Let's open a github issue, describe the issue and use that instead.